packages/frontend-2/components/viewer/PreSetupWrapper.vue

```diff
@@ -314,4 +320,12 @@ watch(
   },
   { immediate: true }
 )
+
+watch(
```

Comment:
this seems to duplicate the existing watcher 
![image](https://github.com/user-attachments/assets/6c0f3e19-55f1-49c7-a43e-200d63742f76)